### PR TITLE
A marginal improvement

### PR DESCRIPTION
**Pull request by CoXier**

### CODE REVIEW

File: matisse/src/main/java/com/zhihu/matisse/Matisse.java [hunk 1]
```diff
@@ -36,8 +36,7 @@
     private final WeakReference<Fragment> mFragment;
 
     private Matisse(Activity activity) {
-        mContext = new WeakReference<>(activity);
-        mFragment = null;
+        this(activity,null);
```
Comment (gejiaheng) [inline matisse/src/main/java/com/zhihu/matisse/Matisse.java]: A space should be placed before parameter `null`

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,6 @@
 *.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
```
Comment (gejiaheng) [inline .gitignore]: Actually, I'm a little bit confused about this removal. I'm not saying you are not right. But can you explain why?

File: .gitignore [hunk 3]
```diff
@@ -25,12 +23,11 @@ bin/
...
-# Gradle files
-.gradle/
-build/
+# Gradle build files
+matisse/build
+sample/build
+
 
-# Local configuration file (sdk path, etc)
-local.properties
```
Comment (gejiaheng) [inline .gitignore]: local properties should be local, right? Say I saved some key and secret in this file, it shouldn't be exposed.

File: .gitignore [hunk 4]
```diff
@@ -25,12 +23,11 @@ bin/
 gen/
 out/
 
-# Gradle files
-.gradle/
-build/
+# Gradle build files
+matisse/build
+sample/build
+
```
Comment (gejiaheng) [inline .gitignore]: So you mean `/Matisse/.gradle` and `/Matisse/build` should not be ignored?

File: .gitignore [hunk 5]
```diff
@@ -44,8 +41,6 @@ proguard/
 # Android Studio captures folder
 captures/
 
-# Intellij
-*.iml
```
Comment (gejiaheng) [inline .gitignore]: Please explain why

Comment (CoXier) [inline .gitignore]: *.iml is appeared in first line.

File: .gitignore (hunk 3 above)
Comment (CoXier) [inline .gitignore]: It's convered by `/local.properties`.

File: matisse/src/main/java/com/zhihu/matisse/Matisse.java (hunk 1 above)
Comment (CoXier) [inline matisse/src/main/java/com/zhihu/matisse/Matisse.java]: Yes.

File: .gitignore (hunk 4 above)
Comment (CoXier) [inline .gitignore]: * .gradle is appeared before.
* `matisse/build` should be ignored not `/Matisse/build`

**Pull request merged**